### ISSUE TITLE
Rendering results of nerf_synthetic lego validation dataset by provided single-scene model

### ISSUE THREAD

**Issue by leo-frank** [VITA-Group/GNT#16]
### **Description:**

I use the pretrained model that you provided at lego synthetic by provided single-scene model at **https://drive.google.com/file/d/1IbhbBr5XfxQz0jSQM3nLX_htTbvc59kj/view?usp=share_link**
the command is: 
`CUDA_VISIBLE_DEVICES=2 python3 eval.py --config configs/gnt_blender.txt \
    --eval_dataset nerf_synthetic \
    --eval_scenes lego  --run_val\
    --expname gnt_author_pretrained_single_lego \
    --ckpt_path out/gnt_lego_from_single_ckpt/pretrained_lego_model_435000.pth \`
 (gnt_blender.txt is not modified)

### **Results:**

1. rendering results on validation set seems not correct, since the color has some green

![image](https://github.com/VITA-Group/GNT/assets/47302141/22999983-a3ac-453f-b38c-fca03605fb1e)

2. but the metrics seems resonable as this screenshot shows:

![image](https://github.com/VITA-Group/GNT/assets/47302141/989a56d8-30d0-4677-8860-02673431d981)

### **Questions:**

I wonder the cause of such unexpected rendering results.Is it caused by my command or anything else?

**Comment by YiftachEde** [comment]
Have you solved this issue? it happens all the same using gnt_full model as a checkpoint model.

**Comment by leo-frank** [comment]
No, I can't solve it. I tried 3 types of model:
1) single-scene model provided by author
2) cross-scene model provided by author
3) single-scene model  trained by myself
all of these have the same issues.

**Comment by alaneze123** [comment]
Hi, have you solved this problem?

**Comment by lan-creator** [comment]
It also happened to me. You can check the RGB value range here:
https://github.com/VITA-Group/GNT/blob/53e0fc3d39b4244503d6c4de1e34bbf3199dd2a8/eval.py#L187-L189

In the original code, there is no value range changed as the rgb_coarse may have a distribution between around [0, 1] and you need to change it to [0, 255] and clip outliers. There is function in utils and we can directly use:
https://github.com/VITA-Group/GNT/blob/53e0fc3d39b4244503d6c4de1e34bbf3199dd2a8/utils.py#L28
